api/build/update-client-gen.sh

```diff
@@ -28,17 +28,20 @@ client-gen "$@" \
 		--go-header-file "${REPO_ROOT}/hack/boilerplate/boilerplate.go.txt" \
 		--input-base "github.com/projectcalico/api/pkg/apis/" \
 		--input "projectcalico/v3" \
+		--output-dir "${REPO_ROOT}/pkg/client/clientset_generated" \
 		--clientset-path "github.com/projectcalico/api/pkg/client/clientset_generated/" \
 		--clientset-name "clientset"
 # generate lister
 lister-gen "$@" \
 		--go-header-file "${REPO_ROOT}/hack/boilerplate/boilerplate.go.txt" \
-		--input-dirs="github.com/projectcalico/api/pkg/apis/projectcalico/v3" \
```

Comment:
`--input-dirs` flag is removed in https://github.com/kubernetes/code-generator/commit/4d6b3bd5ca9ac09408e6e37d251d14ef46c76d0e.